_includes/pbc/all/install-features/202304.0/install-the-shipment-feature.md

```diff
@@ -0,0 +1,341 @@
+     */
+    public function getShipmentDataImporterConfiguration(): DataImporterConfigurationTransfer
+    {
+        return $this->buildImporterConfiguration('shipment.csv', static::IMPORT_TYPE_SHIPMENT);
+    }
+
+    /**
+     * @return \Generated\Shared\Transfer\DataImporterConfigurationTransfer
+     */
+    public function getShipmentMethodPriceDataImporterConfiguration(): DataImporterConfigurationTransfer
+    {
+        return $this->buildImporterConfiguration('shipment_price.csv', static::IMPORT_TYPE_SHIPMENT_PRICE);
+    }
+}
+```
+
+2. Configure shipment GUI module with money and store plugins:
+
+| PLUGIN                            | SPECIFICATION                                                                                              | PREREQUISITES | NAMESPACE                                             |
+|-----------------------------------|------------------------------------------------------------------------------------------------------------|---------------|-------------------------------------------------------|
+| MoneyCollectionFormTypePlugin     | Represents the money collection fields based on stores, currencies, and price types defined in the system. | None          | Spryker\Zed\Money\Communication\Plugin\Form           |
+| StoreRelationToggleFormTypePlugin | Represents a store relation toggle form based on stores registered in the system.                          | None          | Spryker\Zed\Store\Communication\Plugin\Form           |
+| ShipmentTotalCalculatorPlugin     | Calculates shipment total using expenses.                                                                  | None          | Spryker\Zed\Shipment\Communication\Plugin\Calculation |
+
+**src/Pyz/Zed/ShipmentGui/ShipmentGuiDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\ShipmentGui;
+
+use Spryker\Zed\Kernel\Communication\Form\FormTypeInterface;
+use Spryker\Zed\Kernel\Container;
+use Spryker\Zed\Money\Communication\Plugin\Form\MoneyCollectionFormTypePlugin;
+use Spryker\Zed\ShipmentGui\ShipmentGuiDependencyProvider as SprykerShipmentGuiDependencyProvider;
+use Spryker\Zed\Store\Communication\Plugin\Form\StoreRelationToggleFormTypePlugin;
+
+class ShipmentGuiDependencyProvider extends SprykerShipmentGuiDependencyProvider
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Communication\Form\FormTypeInterface
+     */
+    protected function getMoneyCollectionFormTypePlugin(Container $container): FormTypeInterface
+    {
+        return new MoneyCollectionFormTypePlugin();
+    }
+
+    /**
+     * @return \Spryker\Zed\Kernel\Communication\Form\FormTypeInterface
+     */
+    protected function getStoreRelationFormTypePlugin(): FormTypeInterface
+    {
+        return new StoreRelationToggleFormTypePlugin();
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure you can complete the following actions in the Back Office:
```

Comment:
I rewrote the steps below so that the procedure steps follow the google style guide. Ensure that the idea hasn't changed accidentally.